### ISSUE TITLE
Can't Open Profile

### ISSUE THREAD

**Issue by Dowdicus** [WhiteMagic/JoystickGremlin#387]
The last thing I did to my profile was remove the modes it was using.  Now I get an error whenever I start. 
 Every time I try to open and it gives me the error, it leaves an instance of Joystick Gremlin open on my start bar that I can't click on, but I can close it using the task manager.  It's definitely an issue with the profiles that used those modes--I removed them and Joystick Gremlin is able to open and load other profiles.  Is there a way to recover the bad profiles, maybe by editing them in a text file?   Here is my log
[system.log](https://github.com/WhiteMagic/JoystickGremlin/files/7611266/system.log)

**Comment by WhiteMagic** [comment]
By the looks of it there are no modes left and the code needs to have on of those available which is what triggers the error. As to whether or not the profile can be recovered, I'm not sure but I would lean towards no, as I suspect that the everything related to the removed modes has been deleted from the profile already. You can drop the XML in here to see. Also the discord https://discord.gg/szqaJE7 is a useful place for questions like this as others can help out as well in answering them.

**Comment by Dowdicus** [comment]
Okay, thanks.  I am in the discord, idk why I thought it would be better to post here.  I took a look at the profile in a text editor and tried to delete everything related to the old modes, but it still wouldn't open (I suppose I could have missed something).  It won't open my test profile that hasn't had the modes removed yet either (I assume it's still related to the same issue).  I don't suppose it will be too difficult to rebuild the profile.  I guess I can clean it up a little bit in doing so.  I went ahead and attached the two files in a zip.
[Gremlin Profiles.zip](https://github.com/WhiteMagic/JoystickGremlin/files/7612009/Gremlin.Profiles.zip)
